qemu/tests/cfg/block_commit.cfg

```diff
@@ -0,0 +1,34 @@
+# Below case works stablly on QMP monitor, but have no more test with Human
+# monitor, so suggest to use QMP monitor as default qemu monitor run below test
+#
+- block_commit:
+    no raw qed vmdk
+    backup_image_before_testing = yes
+    restore_image_after_testing = yes
+    wait_finished = yes
+    source_image = image1
+    default_speed_image1 = 0
+    snapshot_chain = "images/sn1"
+    # If image size increase or limited committing please increase
+    # wait_timeout to avoid wait for committing finished timeout;
+    wait_timeout = 3900
+    snapshot_format = qcow2
+    kill_vm = yes
+    alive_check_cmd = dir
+    tmp_dir = /tmp
+    variants:
+        - with_stress:
+            type = block_commit_stress
+            download_link = http://fileshare.englab.nay.redhat.com/pub/section2/kvm/linux/stress-1.0.4.tar.gz
```

Comment:
Hi xiyue,

We don't recommend push internal link to upstream.
You can use like "http://FILESHARE" instead.